ihatemoney/web.py

```diff
@@ -760,6 +760,19 @@ def delete_bill(bill_id):
     if not bill:
         return redirect(url_for(".list_bills"))
 
+    # Prevent deleting if the bill involves deactivated user.
+    active_member_id = [m.id for m in g.project.active_members]
+    owers_id = [int(m.id) for m in bill.owers]
+    deactivated_members = set([bill.payer_id] + owers_id).difference(
+        set(active_member_id)
+    )
```

Comment:
I believe it's possible to ask the model for deactivated users, rather than building it yourself here. You need to look for (`activated=False`).

Reply:
Thanks for the suggestion! Will change it.